### ISSUE TITLE
Juno organization

### ISSUE THREAD

**Issue by ChrisRackauckas** [JunoLab/Juno.jl#7]
How exactly is this all organized? For example, where should people leave issues if it's a precompile issue? Or feature requests? Etc. I know that in some sense it might be easier to have it be package specific, but there are many packages, some with little documentation of what part is in them, and they are all interconnected. For the documentation I was think it might be easier to follow what's going on in Julia if in general issues were filed in one place (with developer issues in specific packages). I think that's what you were going for with atom-julia-client but it was never specifically stated. 

So for the docs I was wondering if this could get fleshed out:

- What goes in what repository?
- What issues should go where?
- Where should the docs go?

To me, it seems that if there was a place to have as the "main repository", it would make sense for it to be Juno.jl. Name-wise that's what I would guess. If atom-julia-client will continue to be the main repository, then it should be noted on the README somewhere.

**Comment by pfitzseb** [comment]
We defintely should have some central place to collect all issues from users -- where that is I don't know though... `atom-julia-client` seems to be the one where most users go so far, and we mention discuss.junolab.org in our error messages I think. IMHO we should a) have issues on Github (convenience, organization etc) and b) mention that everywhere: In error messages, the forums and maybe Gitter.
Someone who knows Juno a bit better than the average user can then sort those issues as appropriate (or, better yet, just solve them :D).

Regarding docs: We have the "[manual](https://github.com/JunoLab/atom-julia-client/tree/master/manual)" on github, but if you'll use Documenter.jl for docs in #6 (huge thanks for that!) we should probably switch to that completly and host it on junolab.org.

-
So tl;dr: Let's pick some central repo for issues and feature requests and be done with it. Docs should probably go to junolab.org.

**Comment by MikeInnes** [comment]
I think it's important to distinguish between developer style issues (small, actionable items where we roughly know what needs to happen where) and user support style issues (generally more open-ended things like installation issues or feature requests).

A lot of user support issues can be narrowed down to a dev issue that can obviously go in the relevant repo, but most users don't have the knowledge to do this. I've been pushing discourse for support because it has a lot of advantages in this case:

* Better support for open ended discussion, without worrying about when an issue is closable, as well as moderation when discussions go off the rails (luckily hasn't happened to us yet).
* Doesn't clutter the actionable items within a given repo
* This is huge: posting a new thread on discourse also acts as a simple search, which means users get help quicker and we have to deal with less repetition.
* Not having to think about whether an issue should be left open; closing issues often feels like being shut down for users, even if it makes sense from our perspective because of actionability

The downside is that some people are going to want to just use github, so we may have to choose a "default repo" for issues anyway. Using Juno.jl for that is indeed fitting, since it represents the frontend for users and package authors, and should have fewer actionable issues of its own.

So we can mention this repo as a place for issues but should push the discourse primarily, I think, unless anyone has any strong objections.

**Comment by pfitzseb** [comment]
Makes sense, discourse does seem to be better suited for user issues.
